src/EventsDockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build-env
+WORKDIR /App
+EXPOSE 8080
+
+COPY "Events.sln" "Events.sln"
+COPY "Event/EventAPI/EventAPI.csproj" "Event/EventAPI/EventAPI.csproj"
+COPY "Event/EventApplication/EventApplication.csproj" "Event/EventApplication/EventApplication.csproj" 
+COPY "DAL/DAL.csproj" "DAL/DAL.csproj"
+COPY "API.Abstraction/API.Abstraction.csproj" "API.Abstraction/API.Abstraction.csproj"
+
+RUN dotnet restore "Events.sln"
+# well, following step is a huge overkill but it's 2am I'm exhausted
+COPY . .
```

Comment:
If you did just the what was requested by HomeTask - maybe you wouldn't as exhausted ) Try keeping balance